store/tikv/kv.go

```diff
@@ -276,6 +282,12 @@ func (s *tikvStore) Close() error {
 	if err := s.client.Close(); err != nil {
 		return errors.Trace(err)
 	}
+
+	// For mocktikv, Close() should not only close the client,
+	// but also the underlying store such as leveldb.
+	if raw, ok := s.client.(realClose); ok {
```

Comment:
Check tikvStore.mock == true?

Reply:
Check realClose interface is more precise than mock flag.
If  `tikvStore.mock == true`, maybe it's not leveldb engine and doesn't need to close.
If `client.(realClose)` , RealClose must be called.